apps/site-app/src/containers/App/index.tsx

```diff
@@ -37,30 +39,27 @@ export const App = () => {
   return (
     <MediaContextProvider>
       <style>{mediaStyles}</style>
-      <div id={styles.headerDecoration}>
+
+      <div id={styles.app}>
         <MainNavigation />
-      </div>
 
-      <div className={`${styles.contentContainer}`}>
-        <Container style={{ width: '100%' }} className={``}>
-          <MainPageTransition>
-            <GreaterThanMobileSegment>
-              <Rail internal position='left'>
-                <Sticky context={contextRef}>
-                  <PageSidebar />
-                </Sticky>
-              </Rail>
-            </GreaterThanMobileSegment>
+        <Sidebar />
```

Comment:
Moving sidebar out of it's own grid area means that I have (hopefully) temporarily broken it's sticky nature.

Still worth it, because now it's much easier to control where the sidebar is